lib/blockscore/base.rb

```diff
@@ -5,6 +5,7 @@ class Base
     extend Connection
 
     ABSTRACT_WARNING = 'Base is an abstract class, not an API resource'.freeze
+    API_URL = 'https://api.blockscore.com/'.freeze
```

Comment:
@mbj thanks for your insights, I should have thought of that myself as I did the same change on other code a week ago. Changed.